iree/compiler/Conversion/LinalgToLLVM/LinalgTileAndDistributePass.cpp

```diff
@@ -215,11 +113,39 @@ void LinalgTileAndDistributePass::runOnOperation() {
       }
     });
 
-    TileAndFuseOptions tileAndFuseOptions = {workgroupDistributionOptions,
-                                             allocateThreadLocalMemory};
-    if (failed(tileAndFuseLinalgBufferOps(funcOp, linalgOpsVec, dependenceGraph,
-                                          launchConfig, tileAndFuseOptions))) {
-      return signalPassFailure();
+    if (tiledLoops.empty()) {
+      linalg::LinalgLoopDistributionOptions workgroupDistributionOptions = {
+          [](OpBuilder &builder, Location loc,
+             ArrayRef<Range> parallelLoopRanges) {
+            auto numParallelDims = parallelLoopRanges.size();
+            SmallVector<linalg::ProcInfo, 3> procInfo(numParallelDims);
+            for (size_t dim = 0;
+                 dim < std::min(numParallelDims, static_cast<size_t>(3));
+                 ++dim) {
```

Comment:
I think LLVM style is doing this way: `for (size_t dim = 0, e = ...; dim < e; ++dim`.